src/components/SearchResultsInfo/SearchResultsInfo.tsx

```diff
@@ -0,0 +1,51 @@
+﻿import { Button } from '../Button/Button'
+import { ProfileModel } from '@/data/frontend/profile/types'
+
+import styles from './SearchResultsInfo.module.scss'
+
+interface SearchResultsInfoProps {
+  searchEmailValue: string
+  filteredData: ProfileModel[]
+  clearHandler: () => void
+}
+
+export const SearchResultsInfo = ({
+  searchEmailValue,
+  filteredData,
+  clearHandler,
+}: SearchResultsInfoProps) => {
+  const hasSearchResults = searchEmailValue !== '' && filteredData.length > 0
+  const noSearchResults = searchEmailValue !== '' && filteredData.length === 0
+
+  return (
+    <>
+      {hasSearchResults && (
+        <div className={styles.searchInfoCont}>
+          <p className={styles.searchInfo}>
+            Search results for{' '}
+            <span className={styles.searchValue}>
+              "{searchEmailValue}" ({filteredData.length})
+            </span>
+          </p>
+          <Button variant="action" onClick={clearHandler}>
+            Clear search
+          </Button>
+        </div>
+      )}
+
+      {noSearchResults && (
+        <div className={styles.searchEmptyInfoCont}>
+          <p className={styles.searchInfo}>
+            No search results for{' '}
+            <span className={styles.searchValue}>
+              "{searchEmailValue}" ({filteredData.length})
+            </span>
+          </p>
+          <Button variant="primary" onClick={clearHandler}>
+            Clear search
+          </Button>
+        </div>
+      )}
+    </>
+  )
+}
```

Comment:
We adhere to the DRY(Don't Repeat Yourself) principle, so you can simply use:
 "const isSearchActive = searchEmailValue !== '';" 
 
 and then conditionally render the classes: 
 "{isSearchActive && (
      <div className={styles[filteredData.length > 0 ? 'searchInfoCont' : 'searchEmptyInfoCont']}>..."
      
      By doing this, the code is halved in length